### PR TITLE
Add papertrail

### PR DESCRIPTION
**Pull request by cisaacstern**
@andersy005 and I realized that Heroku server logs are quite limited without an add-on. This PR adds the [Papertrail add-on](https://elements.heroku.com/addons/papertrail) for logs retention/filtering.

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
     "review": {
       "env": {
         "PANGEO_FORGE_DEPLOYMENT": "pforge-pr-136"
-      }
+      },
+      "addons": ["papertrail:choklad"]
```
Comment (andersy005) [inline app.json:18]: does this mean this is only enabled for review apps?

Comment (cisaacstern) [reply app.json:18]: @andersy005, yes I'm going to start by seeing what this looks like in a review app, but if it works as I'd hope, I will add it to the top-level add-ons before merging this PR.

Comment (cisaacstern) [reply app.json:18]: I figured out that you could configure review app add-ons by reading this https://devcenter.heroku.com/articles/github-integration-review-apps#environments-in-app-json

**Comment by cisaacstern** [comment]
@andersy005, https://github.com/pangeo-forge/pangeo-forge-orchestrator/pull/143/commits/585f10fdb4a9d449865636c169611c1a13529b69 is that (overly complex) process I was telling you about for setting up review app config on the current PR. Again, you may not need/want to deal with this, but this is what I've been doing, FWIW. It would be great to streamline/automate this eventually, but for now this is what it is.

**Comment by cisaacstern** [comment]
Following on https://github.com/pangeo-forge/pangeo-forge-orchestrator/pull/143#issuecomment-1260251612, to complete the review app setup on GitHub, I also had to update it's name and webhook url (this is from the `pforgetest` org's GitHub App settings page):

Screen Shot 2022-09-27 at 5 53 20 PM

Screen Shot 2022-09-27 at 5 53 15 PM


Finally, on the Heroku Pipelines dashboard, I deleted the review app, and then re-created (re-built) it from scratch:

Screen Shot 2022-09-27 at 5 55 55 PM

This last step is consistent with the advice in:

https://github.com/pangeo-forge/pangeo-forge-orchestrator/blob/af5da75d421ffb069b92f48a61b882edef8b9f3b/docs/README.md?plain=1#L396-L403

**Pull request merged**